packages/platform-browser/src/dom/dom_renderer.ts

```diff
@@ -396,6 +396,7 @@ function checkNoSyntheticProp(name: string, nameKind: string) {
       RuntimeErrorCode.UNEXPECTED_SYNTHETIC_PROPERTY,
       `Unexpected synthetic ${nameKind} ${name} found. Please make sure that:
   - Either \`BrowserAnimationsModule\` or \`NoopAnimationsModule\` are imported in your application.
+  - If you use standalone API, make sure \` provideAnimationsAsync() \` or \` provideAnimations() \` was added to application config in app.config.ts
   - There is corresponding configuration for the animation named \`${name}\` defined in the \`animations\` field of the \`@Component\` decorator (see https://angular.io/api/core/Component#animations).`,
```

Comment:
I'd propose to drop references to `BrowserAnimationsModule` and `NoopAnimationsModule` and refer to standalone functions (which can also be used for NgModule-based bootstrap):
```suggestion
  - Make sure \` provideAnimationsAsync()\`, \` provideAnimations()\` or \`provideNoopAnimations()\` call was added to a list of providers used to bootstrap an application.
  - There is a corresponding animation configuration named \`${name}\` defined in the \`animations\` field of the \`@Component\` decorator (see https://angular.dev/api/core/Component#animations).`,
```


Unrelated (outside of the scope of this PR): we should probably deprecate `BrowserAnimationsModule` and `NoopAnimationsModule` in favor of ` provideAnimationsAsync() `, ` provideAnimations()` and `provideNoopAnimations()`. @thePunderWoman what do you think?

Reply:
Yeah, I'm definitely on board with that plan. Let's do it.

Reply:
Sorry, I completely forgot about my PR :D 
@AndrewKushnir agree! I updated the error message.